Lib/http/cookiejar.py

```diff
@@ -50,10 +50,18 @@ def _debug(*args):
         logger = logging.getLogger("http.cookiejar")
     return logger.debug(*args)
 
-
+HTTPONLY_ATTR = "HTTPOnly"
```

Comment:
This implementation is similar to my alternative approach in https://github.com/python/cpython/pull/22798 (and see bpo-2190), _except_:

- This version:
  - **Check if a cookie is "HTTP-only"** with `cookie.has_nonstandard_attr(http.cookiejar.HTTPONLY_ATTR'))`
  - **Set a cookie as "HTTP-only"** with `cookie.set_nonstandard_attr("HTTPOnly", "")`
- My version: 
  - **Check** with `cookie.httponly`.
  - **Set** with `cookie.httponly = True`